tests/cypress/e2e/recover_page/modal_alert_popup.cy.js

```diff
@@ -0,0 +1,34 @@
+import {goToRecover} from '../tools/navigation/nav.js'
+
+describe('Modal alert popup test', () => {
+  it('passes', () => {
+    goToRecover();
+
+    // Tests hidden modal alert text
+    cy.get('#modal_alert_text')
+      .should('not.be.visible')
+      .should('be.empty');
+
+    // Tests visibility of error modal alert text/ recover non existing account
+    cy.get('#username')
+      .type('anonexistingaccount123@#$%^!')
+    cy.get('button[class*="green-btn mt-2"]')
+       .click()
+    cy.get('#modal_alert_text')
+      .should('be.visible')
+      .contains('Your username is invalid.');
+
+    // Reset fields
+    cy.get('#username')
+      .clear();
+
+    // Tests visibility of succes modal alert text/ recover existing account
+    cy.get('#username')
+      .type('user1')
+    cy.get('button[class*="green-btn mt-2"]')
+       .click()
+    cy.get('#modal_alert_text')
+      .should('be.visible')
+      .contains('You should soon receive an email with instructions on how to reset your password.');
```

Comment:
This is also a tricky one because of the language of the user or what if we decide to change the string a bit? I understand that you need values and attributes to validate, but try to fallback on "robust" ones that are actually determining the functionality as much as possible.